packages/features/ee/managed-event-types/reassignment/services/ManagedEventManualReassignmentService.ts

```diff
@@ -0,0 +1,882 @@
+          createdEvent?.url ||
+          getVideoCallUrlFromCalEvent(evt) ||
+          videoCallUrl;
+      }
+
+      videoCallData = evt.videoCallData;
+
+      try {
+        const responses = {
+          ...(typeof newBooking.responses === "object" && newBooking.responses),
+          location: {
+            value: bookingLocation,
+            optionValue: "",
+          },
+        };
+
+        const finalVideoCallUrlForMetadata = videoCallUrl
+          ? getVideoCallUrlFromCalEvent(evt) || videoCallUrl
+          : null;
+        const bookingMetadataUpdate = finalVideoCallUrlForMetadata
+          ? { videoCallUrl: finalVideoCallUrlForMetadata }
+          : {};
+
+        const referencesToCreateForDb = referencesToCreate.map((reference) => {
+          const { credentialId, ...restReference } = reference;
+          return {
+            ...restReference,
+            ...(credentialId && credentialId > 0 ? { credentialId } : {}),
+          };
+        });
+
+        await this.bookingRepository.updateLocationById({
+          where: { id: newBooking.id },
+          data: {
+            location: bookingLocation,
+            metadata: {
+              ...(typeof newBooking.metadata === "object" && newBooking.metadata
+                ? newBooking.metadata
+                : {}),
+              ...bookingMetadataUpdate,
+            },
+            referencesToCreate: referencesToCreateForDb,
+            responses,
+            iCalSequence: (newBooking.iCalSequence || 0) + 1,
+          },
+        });
+
+        logger.info("Updated booking location and created calendar references", {
+          referencesCount: referencesToCreate.length,
+          videoCallUrl: bookingMetadataUpdate.videoCallUrl ? "[redacted]" : null,
+        });
+      } catch (error) {
+        logger.error("Error updating booking location and references", error);
+      }
+
+      logger.info("Created calendar events for new user");
+    } catch (error) {
+      logger.error("Error creating calendar events for new user", error);
+    }
+
+    return {
+      bookingLocation,
+      videoCallUrl,
+      videoCallData,
+      additionalInformation,
+    };
+  }
+
+  /**
+   * Cancels workflow reminders for the original booking
+   */
+  private async cancelExistingWorkflows({
+    originalBookingFull,
+    logger,
+  }: {
+    originalBookingFull: NonNullable<
+      Awaited<ReturnType<BookingRepository["findByIdWithAttendeesPaymentAndReferences"]>>
+    >;
+    logger: typeof logger;
+  }) {
+    try {
+      const cancelResult = await this.workflowReminderService.cancelWorkflowReminders(
+        originalBookingFull.workflowReminders
+      );
+      logger.info(`Cancelled ${cancelResult.cancelledCount} workflow reminders`);
+    } catch (error) {
+      logger.error("Error cancelling workflow reminders", error);
+    }
+  }
+
+  /**
+   * Schedules workflow reminders for the new booking
+   */
+  private async scheduleWorkflowsForNewBooking({
```

Comment:
Same here, nothing specific to managed event types, we can call the Workflow service function directly